clients/unshaded/src/main/java/tachyon/client/table/RawTableMasterClient.java

```diff
@@ -75,7 +77,8 @@ public synchronized long createRawTable(final TachyonURI path, final int columns
     return retryRPC(new RpcCallableThrowsTachyonTException<Long>() {
       @Override
       public Long call() throws TachyonTException, TException {
-        return mClient.createRawTable(path.getPath(), columns, metadata);
+        RpcOptions rpcOptions = new RpcOptions().setKey(IdUtils.createRpcId());
```

Comment:
Sure, I've opened a JIRA for this issue: https://tachyon.atlassian.net/browse/TACHYON-1309